apps/web/pages/auth/login.tsx

```diff
@@ -94,17 +103,22 @@ export default function Login({
     </Button>
   );
 
+  const communityLogin = queryParams.sso !== undefined && queryParams.sig !== undefined;
+
   const onSubmit = async (values: LoginValues) => {
     setErrorMessage(null);
     telemetry.event(telemetryEventTypes.login, collectPageParameters());
     const res = await signIn<"credentials">("credentials", {
       ...values,
-      callbackUrl,
-      redirect: false,
+      callbackUrl: communityLogin
+        ? `${COMMUNITY_URL}?sso=${queryParams.sso}&sig=${queryParams.sig}`
+        : queryParams.callbackUrl,
+      redirect: communityLogin,
     });
+    if (!res && communityLogin) return setSuccessMessage("Success! Redirecting...");
```

Comment:
When community login succeeds, then no `res` is returned and when that happens in the context of a community login, everything went well, so showing message while being redirected.

Reply:
Oh I guess this string needs to be localized 😅